api/v1alpha1/nats_types.go

```diff
@@ -125,6 +125,7 @@ type Cluster struct {
 	// +kubebuilder:default:=3
 	// +kubebuilder:validation:Minimum:=1
 	// +kubebuilder:validation:XValidation:rule="(self%2) != 0", message="size only accepts odd numbers"
+	// +kubebuilder:validation:XValidation:rule="!(oldSelf > 1 && self == 1)", message="size cannot be reduced to 1 once it was >1"
```

Comment:
No, the rule is, once it was set to greater than `1` it can never be `1` again. I can try to make the message clearer like `cannot be set to 1 if size was greater than 1`?

Reply:
Okay, looks I got that wrong. Nevermind then.